app/livechat/server/methods/setUpConnection.js

```diff
@@ -15,6 +15,7 @@ Meteor.methods({
 			this.connection.livechatToken = token;
 			this.connection.onClose(() => {
 				Livechat.notifyGuestStatusChanged(token, 'offline');
+				Livechat.notifyGuestSessionStatusChanged(token, 'offline');
```

Comment:
Why do you need the `notifyGuestSessionStatusChanged` if you already have the `notifyGuestStatusChanged` ?